packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -2481,6 +2528,13 @@ describe('plugin-meetings', () => {
             receiveVideo: true,
             receiveShare: true,
           }
+          meeting.sendSlotManager = new SendSlotManager({
```

Comment:
why is this needed? doesn't the meeting already have a SendSlotManager instance? it should be created when the meeting object is created, right?

Reply:
Makes sense. Removed all creation of SendSlotManager. 

In fact we also have tests to ensure that the SendSlotManager is created as part of meeting creation. Thanks for pointing out.